NuRadioMC/examples/05_pulser_calibration_measurement/radioEmitting_pulser_calibration/config.yaml

```diff
@@ -0,0 +1,15 @@
+noise: False  # specify if simulation should be run with or without noise
+sampling_rate: 2.  # sampling rate in GHz used internally in the simulation.
+speedup:
+  minimum_weight_cut: 1.e-5
+  delta_C_cut: 100  # 40 degree
+  redo_raytracing: True  # redo ray tracing even if previous calculated ray tracing solutions are present
+  distance_cut: False # Skip ray tracing if vertex distance is greater than a distance cut that is a function of shower energy.
+propagation:
+  ice_model: southpole_2015
+signal:
```

Comment:
as you don't use an Askaryan model ,better remove this part of the config file to avoid confusions. Same for the `weights` category.